renderer/src/pages/WalletConfig.tsx

```diff
@@ -1,33 +1,18 @@
-import { useState, useEffect, useCallback } from 'react'
+import { useEffect, useCallback } from 'react'
 import FilAddressForm from '../components/FilAddressForm'
 import BackgroundGraph from './../assets/img/graph.svg'
 import { useNavigate } from 'react-router-dom'
-import { getFilAddress, setFilAddress as saveFilAddress } from '../lib/station-config'
+import { setFilAddress as saveFilAddress } from '../lib/station-config'
 
 const WalletConfig = (): JSX.Element => {
   const navigate = useNavigate()
-
-  const [filAddress, setFilAddress] = useState<string | undefined>(undefined)
-
   useEffect(() => { document.title = 'Login' })
```

Comment:
this should have been removed 🤔  Does this PR have `main` merged in cleanly?

Reply:
[bbd4874](https://github.com/filecoin-project/filecoin-station/pull/144/commits/bbd4874fb7d2c3334f0ba0cca3eaaba70f97b899)